### DISCUSSION TITLE
extract datas in a form (excel or anything else)

### DISCUSSION THREAD

**Discussion by Benjamin68650** [opengisch/QField#5042] [Q&A]
Hi,
I'm looking for a solution, in order to "extract" my QFIELD datas in an Excel table.
More precisely, I don't wan't to open a SHAPE in QGIS, and copy/paste the datas in excel. 
I need an easier solution, if possible directely in QFIELD.

Do you think it's possible ?
Thanks !

**Comment by VxTedxV** [comment]
Hi,
In Qgis you can export your layer directly in Excel format (Layer > Save as... > Format : MS Office Open XML spreadsheet [XLSX])
You can either use an existing export script (in Qgis, Toolbox > Layer tools > Export to spreadsheet) or create yours to be more effective (and use it with a custom button).
In QField, you can mainly load and gather data. You can just export in PDF (more graphic maps than raw datas but remain doable).

**Reply by Benjamin68650** [reply]
thanks a lot ;-)
